pkgs/servers/pulseaudio/default.nix

```diff
@@ -57,6 +61,9 @@ stdenv.mkDerivation rec {
     ./0006-Only-use-version-script-on-GNU-ish-linkers.patch
     ./0007-Adapt-undefined-link-args-per-linker.patch
     ./0008-Use-correct-semaphore-on-darwin.patch
+    ./0009-Do-not-include-cpuid.h-on-aarch64-darwin.patch
+    ./0010-Enable-coreaudio-on-darwin.patch
+    ./0011-Patch-link-args-on-darwin.patch
```

Comment:
Can we move some of those patches out of this conditional to not break it when updating pulseaudio on linux?

Reply:
Yeah, I think, if we don't really care about massive Linux rebuilds. I will rewrite the patches and test in my repo.

Reply:
yeah, we can just send it to staging like we already do. Right now we could even send it to master since the builds are all new.

Reply:
No more darwin-only patches. Tested on all four platforms. I guess the commit will have to stay in staging now.